cfme/scripting/setup_env.py

```diff
@@ -206,5 +210,93 @@ def setup_replication_env(cfme_version, provider_type, provider, lease, sprout_p
     print("Done!")
 
 
+@main.command('multi-region', help='Sets up replicated environment')
+@click.option('--cfme-version', required=True)
+@click.option('--provider-type', default='rhevm', help='Specify sprout provider type')
+@click.option('--provider', default=None, help='Specify sprout provider, overrides provider_type')
+@click.option('--lease', default='3d', help='set pool lease time, example: 1d4h30m')
+@click.option('--sprout-poolid', default=None, help='Specify ID of existing pool')
+@click.option('--desc', default='Replicated appliances', help='Set description of the pool')
+@click.option('--remote-nodes', default=2, type=int, help='Add nodes to remote regions')
+@click.option('--add-prov', default=None, type=str, multiple=True,
+              help='Add providers to remote region appliances')
+def setup_multiregion_env(cfme_version, provider_type, provider, lease, sprout_poolid, desc,
+                          remote_nodes, add_prov):
+    lease_time = tot_time(lease)
+    provider_type = None if provider else provider_type
+    """Multi appliance setup with multi region and replication from remote to global"""
+
+    sprout_client = SproutClient.from_config()
+
+    required_app_count = 1  # global app
+    required_app_count += remote_nodes
+
+    if sprout_poolid:
+        if sprout_client.call_method('pool_exists', sprout_poolid):
+            sprout_pool = sprout_client.call_method('request_check', sprout_poolid)
+            if len(sprout_pool['appliances']) >= required_app_count:
+                print("Processing pool...")
+                apps = []
+                for app in sprout_pool['appliances']:
+                    apps.append(IPAppliance(app['ip_address']))
+                sprout_client.set_pool_description(sprout_poolid, desc)
+            else:
+                sys.exit("Pool does not meet the minimum size requirements!")
+        else:
+            sys.exit("Pool not found!")
+
+    else:
+        print("Provisioning appliances")
```

Comment:
I got a comment about using `click.echo` rather than `print` on one of my PR's a while back. Some info on the difference is here: https://stackoverflow.com/questions/49906680/difference-between-print-and-click-echo-in-python-3. @digitronik is it worth changing `print` to `click.echo` here?

Reply:
this is good improvement but it is out of scope. I follow existing code style. 
if we're going to use `click.echo` instead of `print`, we will have to update all our scripting stuff.

Reply:
Agreed, if we want to move to `click.echo`, we should do it in its own PR.